main.go

```diff
@@ -144,9 +142,10 @@ func run(ctx context.Context, args []string, errOutput io.Writer, flagSet *flag.
 			return WriteProfile(profile, outF, funcMap)
 		}))
 	}
-	if *coverageProfile != "" {
+	if os.Getenv("GOCOVERDIR") != "" {
 		tasks = append(tasks, chromedp.ActionFunc(func(ctx context.Context) error {
-			return os.WriteFile(*coverageProfile, []byte(coverageProfileContents), 0644)
+			// TODO: maybe future covmeta, covcounter file copies
```

Comment:
This was in reference to [working with coverage data files](https://go.dev/testing/coverage/#working). 
i.e. Being able to use the covmeta and covcounter files with the `go covdata`.

I just figured out that `-test.gocoverdir=/path/to/dir` can be used instead, so this can be removed.